incubator/haproxy-ingress/values.yaml

```diff
@@ -239,9 +237,23 @@ controller:
     # (scrapes the stats port and exports metrics to prometheus)
     image:
       repository: quay.io/prometheus/haproxy-exporter
-      tag: "v0.9.0"
+      tag: "v0.10.0"
       pullPolicy: IfNotPresent
 
+    ## Additional command line arguments to pass to haproxy_exporter
+    ## E.g. to specify the client timeout you can use
+    ## extraArgs:
+    ##   haproxy.timeout: 15s
+    extraArgs: {}
+
+    resources: {}
+    #  limits:
```

Comment:
Maybe  we should add some default value to this limit

Reply:
@xianlubird I wasn't sure what was best here. There is no default set for for the main container, so I decided not to provide defaults for the side-car containers either. The prometheus-exporter container resources depend a bit on the haproxy config and how many ingresses (backends/servers) there are. We had to increase the resources when we deployed more services, because the number of metrics increased.

Anyway, the commented-out resources in values.yaml are what we are currently using in production. If you want I can uncomment them and provide them as defaults.

Reply:
That make sense ,  thanks.